src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1274,6 +1320,22 @@ def _public_key_bytes(self, encoding, format, evp_pkey, cdata):
         assert res == 1
         return self._read_mem_bio(bio)
 
+    def _parse_asn1_time(self, asn1_time):
```

Comment:
I recently moved similar methods to free functions and renamed them to `_decode_*`.  Maybe you can align with that?